.editorconfig

```diff
@@ -0,0 +1,12 @@
+# http://EditorConfig.org
+
+# top-level EditorConfig file
+root = true
+
+[*]
+indent_style = space
+charset = utf-8
+
+[*.{ts,tsx,js,json}]
+indent_size = 2
+end_of_line = lf
```

Comment:
```suggestion
[*]
charset = utf-8
end_of_line = lf
indent_size = 2
indent_style = space
insert_final_newline = true
trim_trailing_whitespace = true
```

Reply:
I think this will affect binary files and other files that we may not want to modify for whatever reason.

Reply:
no no.. it only affects text files.. I'm using it for years like that.

Reply:
Cool!